FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2AppBarActivity.kt

```diff
@@ -197,6 +202,23 @@ class V2AppBarActivity : V2DemoActivity() {
                                 )
                             }
                         )
+
+                        ListItem.Item(
+                            text = LocalContext.current.resources.getString(R.string.app_bar_bottom_border),
+                            subText = if (enableBottomBorder)
+                                LocalContext.current.resources.getString(R.string.fluentui_enabled)
+                            else
+                                LocalContext.current.resources.getString(R.string.fluentui_disabled),
+                            trailingAccessoryContent = {
+                                ToggleSwitch(
+                                    onValueChange = {
+                                        enableBottomBorder = !enableBottomBorder
+                                    },
+                                    modifier = Modifier.testTag(APP_BAR_SEARCHBAR_PARAM),
```

Comment:
use different  testTag as it fails the test case

Reply:
Done